dev/docker-compose.yaml

```diff
@@ -102,6 +154,8 @@ services:
       # NB accessControlAllowOrigin is deprecated, but not noted in docs
       # https://github.com/traefik/traefik/issues/8796
       - "traefik.http.middlewares.fhirwall-${COMPOSE_PROJECT_NAME}-cors.headers.accessControlAllowOriginList=*"
+      # allow all verbs used by FHIR REST
+      - "traefik.http.middlewares.fhirwall-${COMPOSE_PROJECT_NAME}-cors.headers.accessControlAllowMethods=HEAD,GET,OPTIONS,PATCH,POST,PUT,DELETE"
```

Comment:
I think these CORS header changes are only necessary if we expect the frontend to talk directly to `fhirwall` (ala ISACC) vs through the sof-backend fhir-router.

Reply:
wow, great catch.  thank you.

Reply:
on second thought, we have both confidential AND public clients included in this demo.  let's keep this for now as it is required by the public ones.